app/javascript/packages/test-helpers/use-sandbox.ts

```diff
@@ -22,12 +22,32 @@ function useSandbox(config?: Partial<SinonSandboxConfig>) {
     sandbox.clock.restore();
   }
 
+  // useFakeTimers overrides global.setTimeout, etc. (callable as setTimeout()), but does not
```

Comment:
FYI @aduth this is fun fallout from the `setTimeout` vs `window.setTimeout` issue. Sinon replaces `global.setTimeout` but NOT `window.setTimeout`. There is a config option to set the `global` it uses, but I think that would break any existing "bare" references to `setTimeout`.

Reply:
Gotcha, this makes sense then 👍 

Alternatively, I wonder what might happen if we changed the "window" `setTimeout` to point to the Node global in the test setup?

```ts
window.setTimeout = setTimeout;
```